src/Generator.Bind/Main.cs

```diff
@@ -57,7 +57,13 @@ private static void Main(string[] arguments)
                     if (match.Success)
                     {
                         string opt = match.Value.Substring(1).Trim();
-                        string val = a.Substring(match.Value.Length + 1).Trim();
+                        string val;
+
+                        if (a.Length != match.Value.Length)
+                            val = a.Substring(match.Value.Length + 1).Trim();
+                        else
+                            val = string.Empty;
```

Comment:
Not sure why we are just checking to see if the lengths are different, I think we should decide on a separator and check for it explicitly.

Maybe using a regex like this and using the captures: 
```regex
-(\w+)(:(.+))?
```
or just doing a check like:
```cs
if (a.length > match.Value.Length && a[match.Value.Length] == ':')
{
// the rest...
```
I think we should use `>` instead of `!=` for the string lengths.

Reply:
How about using Contains in if statement and substring with IndexOf as below demonstration?
```
    if (a.Contains(":"))
        val = a.Substring(a.IndexOf(':') + 1).Trim();
```

Reply:
Sounds reasonable enough. 
A little less verification then the regex idea but I'm fine with it.